spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2436Test.java

```diff
@@ -0,0 +1,24 @@
+package edu.umd.cs.findbugs.detect;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.annotations.Confidence;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.Test;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class Issue2436Test extends AbstractIntegrationTest {
+
+    @Test
+    public void test() {
+        performAnalysis("ghIssues/Issue2436.class");
+        BugInstanceMatcher bugMatcher = new BugInstanceMatcherBuilder()
+                .bugType("SE_NO_SERIALVERSIONID")
+                .withConfidence(Confidence.LOW)
+                .build();
```

Comment:
There are usually two ways when we use BugInstanceMatcher:
1. negative tests: the BugInstanceMatcher should contain minimal info (only the bugtype, maybe the containing class), used when we want to assert the number of hits of a certain bugtype, or we want to check that a bug is not reported at all.
2. positive tests: the BugInstanceMatcher should contain maximal info, used when we want to assert that a bug is reported in a certain way exactly.

IMO adding the confidence makes it closer to the positive tests, than to the negative, general ones.

Reply:
ok, got it